integration_tests/src/util/workflows.rs

```diff
@@ -209,6 +210,46 @@ pub struct PrecommitMetadata {
     pub commd: CompactCommD,
 }
 
+pub fn build_verifreg_event(
+    typ: &str,
+    id: u64,
+    client: ActorID,
+    provider: ActorID,
+) -> EmittedEvent {
+    EmittedEvent {
+        emitter: VERIFIED_REGISTRY_ACTOR_ID,
+        event: EventBuilder::new()
+            .typ(typ)
+            .field_indexed("id", &id)
+            .field_indexed("client", &client)
+            .field_indexed("provider", &provider)
+            .build()
+            .unwrap(),
+    }
+}
+pub fn build_market_event(typ: &str, provider: ActorID, deal_id: DealID) -> EmittedEvent {
+    EmittedEvent {
+        emitter: provider,
+        event: EventBuilder::new().typ(typ).field_indexed("id", &deal_id).build().unwrap(),
+    }
+}
+
+pub fn build_miner_event(
+    typ: &str,
+    miner_id: ActorID,
+    sector_number: SectorNumber,
+) -> EmittedEvent {
+    EmittedEvent {
+        emitter: miner_id,
+        event: EventBuilder::new()
+            .typ(typ)
+            .field_indexed("provider", &miner_id)
```

Comment:
This makes clear the point @Stebalien made elsewhere that the miner ID is redundant here, if it's the emitter too. So let's remove it.

Reply:
Yes, this makes sense. Done.